sdk/src/test/java/com/horizen/account/transaction/EthereumTransactionTest.java

```diff
@@ -549,7 +215,7 @@ public void ethereumTransactionsRootHashTest() {
     @Test
     public void ethereumTransactionDecoderTest() {
         // Test 1: Decoded tx should be as expected - same tx as linked above, just without access list
-        var actualTx = new EthereumTransaction(EthereumTransactionDecoder.decode("0x02f8c60183012ec786023199fa3df88602e59652e99b8303851d9400000000003b3cc22af3ae1eac0440bcee416b4080b8530100d5a0afa68dd8cb83097765263adad881af6eed479c4a33ab293dce330b92aa52bc2a7cd3816edaa75f890b00000000000000000000000000000000000000000000007eb2e82c51126a5dde0a2e2a52f701c080a020d7f34682e1c2834fcb0838e08be184ea6eba5189eda34c9a7561a209f7ed04a07c63c158f32d26630a9732d7553cfc5b16cff01f0a72c41842da693821ccdfcb"));
+        var actualTx = EthereumTransactionDecoder.decode("0x02f8c60183012ec786023199fa3df88602e59652e99b8303851d9400000000003b3cc22af3ae1eac0440bcee416b4080b8530100d5a0afa68dd8cb83097765263adad881af6eed479c4a33ab293dce330b92aa52bc2a7cd3816edaa75f890b00000000000000000000000000000000000000000000007eb2e82c51126a5dde0a2e2a52f701c080a020d7f34682e1c2834fcb0838e08be184ea6eba5189eda34c9a7561a209f7ed04a07c63c158f32d26630a9732d7553cfc5b16cff01f0a72c41842da693821ccdfcb");
         var expectedTx = new EthereumTransaction(1, "0x00000000003b3cc22aF3aE1EAc0440BcEe416B40", Numeric.toBigInt("12EC7"), Numeric.toBigInt("0x03851d"), Numeric.toBigInt("0x023199fa3df8"), Numeric.toBigInt("0x02e59652e99b"), BigInteger.ZERO, "0x0100d5a0afa68dd8cb83097765263adad881af6eed479c4a33ab293dce330b92aa52bc2a7cd3816edaa75f890b00000000000000000000000000000000000000000000007eb2e82c51126a5dde0a2e2a52f701", null);
         assertArrayEquals(expectedTx.messageToSign(), actualTx.messageToSign());
```

Comment:
Maybe a check if the 2 txs are equal could be added. In addition, maybe a test with a legacy transaction could be added too.

Reply:
Done